src/grpc/client.ts

```diff
@@ -1,11 +1,11 @@
+import { ChannelCredentials } from "@grpc/grpc-js";
 import { GrpcTransport } from "@protobuf-ts/grpc-transport";
 import { LoggerAPIClient } from "./generated/logger/v1alpha1/log.client";
-import { ChannelCredentials } from "@grpc/grpc-js";
 
 export function createLoggerClient(): LoggerAPIClient {
-        const transport = new GrpcTransport({
-                host: "grid-logger.deepsquare.run:443",
-                channelCredentials: ChannelCredentials.createSsl(),
-        });
-        return new LoggerAPIClient(transport);
+  const transport = new GrpcTransport({
+    host: "grid-logger.deepsquare.run:443",
```

Comment:
Shouldn't this be an environment variable ?

Reply:
It's already a parameter, you can inject a factory function inside the `build` function.